client/src/components/Onboarding/OnboardingAdmin.js

```diff
@@ -294,7 +294,11 @@ const OnboardingAdmin = () => {
   }
   if (error) {
     return (
-      <Message type="warn">Unable to access Onboarding administration</Message>
+      <div styleName="root-placeholder">
+        <Message type="warn">
+          Unable to access Onboarding administration
+        </Message>
```

Comment:
While you're in here, any interest in upgrading this to a "section" message?

```suggestion
        <SectionMessage type="warn">
          Unable to access Onboarding administration
        </SectionMessage>
```

Requires adding `SectionMessage` to the import list that has `Message`:

<details>
<summary>Related Complementary Code Change</summary>

```
import {
  Icon,
  LoadingSpinner,
  SectionMessage,
  Message,
  Paginator
} from '_common';
```

</details>

<details>
<summary>Screenshot</summary>

![Onboarding SectionMessage](https://user-images.githubusercontent.com/62723358/119698183-9dbe8980-be16-11eb-8a88-485c95542f48.png)

</details>